gems/yrb-actioncable/lib/y/lib0/decoding.rb

```diff
@@ -28,6 +28,8 @@ def self.clone(decoder, new_pos = decoder.pos)
 
       def self.read_uint8_array(decoder, size)
         view = Buffer.create_uint8_array_view_from_buffer(decoder.arr, decoder.pos + 0, size)
+        decoder.pos += size
+        view
```

Comment:
Even better, those warning disappears.


at main branch:

```
$ ruby -cw gems/yrb-actioncable/lib/y/lib0/decoding.rb
gems/yrb-actioncable/lib/y/lib0/decoding.rb:30: warning: assigned but unused variable - view
Syntax OK
```


now this PR branch:

```
$ ruby -cw gems/yrb-actioncable/lib/y/lib0/decoding.rb
Syntax OK
```